### DISCUSSION TITLE
Add a customizable featured project label

### DISCUSSION THREAD

**Discussion by bobthecowinc** [ScratchAddons/ScratchAddons#2913] [Ideas] [answered]
Please maybe add a feature so that i can make my 'featured project' label, not just the things they have, like work in progress, remix this, and why I scratch. An actual customizable label please.

**Comment by cobaltt7** [comment, marked as answer]
impossible sorry

**Comment by mxmou** [comment]
There isn't a way to make non-SA users see the label, but would this implementation be good enough?
- The label and link to the project is added to your About me or What I'm Working On
- People who don't use Scratch Addons will see the featured project normally, but they will also be able to click the link with the custom label
- When a Scratch Addons user visits your profile, Scratch Addons would look for a link to your featured project and replace the label with the text next to the first link it finds

**Reply by WorldLanguages** [reply]
Eh, too complex for little benefit
